autofill/autofill-impl/src/main/java/com/duckduckgo/autofill/impl/importing/InBrowserPromoPreviousPromptsStore.kt

```diff
@@ -48,10 +48,15 @@ class InMemoryInBrowserPromoPreviousPromptsStore @Inject constructor(
             eTldPlusOne.equals(previousETldPlusOneOffered, ignoreCase = true)
         }
     }
+
+    override suspend fun clearPreviousPrompts() {
+        previousETldPlusOneOffered = null
+    }
 }
 
 interface InBrowserPromoPreviousPromptsStore {
 
     suspend fun recordPromoDisplayed(originalUrl: String)
     suspend fun hasPromoBeenDisplayed(originalUrl: String): Boolean
+    suspend fun clearPreviousPrompts()
```

Comment:
I don't think we should add methods into real interfaces exclusively for internal testing. This seems to exist only for internal testing purposes, can we remove it? or find another way if we need it?

(it also refers to previous prompts but it's clearing a previousETld cache, naming seems not accurate)

Reply:
I can separate into a _public_ and _internal_ API like we do for `AutofillStore` (and rename)